daml/splice-dso-governance/daml/Splice/DsoRules.daml

```diff
@@ -1785,6 +1846,32 @@ template UnallocatedUnclaimedActivityRecord
     signatory dso
     ensure amount > 0.0
 
+-- Vote indirections
+--------------------
+
+-- Vote requests fix the choice arguments. This doesn't work well for choices that need to act on
+-- contracts that get cycled frequently, e.g., ValidatorLicense. The contracts here act as an indirection to handle that:
+-- The vote is on a choice that creates one of these contracts. Automation then uses those contracts as authorization
+-- to exercise the actual action.
```

Comment:
makes sense. good name: Instruction

Reply:
@meiersi-da Not very sure, but it be better to have a single template with the various instruction types part of it, (similar to `action : ActionRequiringConfirmation` ). That would make the scala triggers simpler. Having separate template would require one trigger per instruction type.